src/stores/Create.ts

```diff
@@ -308,20 +410,23 @@ export const getInitialCreateState = (
         },
 
         hasChanges: () => {
-            const { details, endpointConfig } = get();
+            const { details, endpointConfig, resourceConfig } = get();
             const {
                 details: initialDetails,
-                endpointConfig: initialendpointConfig,
+                endpointConfig: initialEndpointConfig,
+                resourceConfig: initialResourceConfig,
```

Comment:
Checking resource config. This should honestly be moved to a computed value as well. Will handle that later.